test/reload_config.bats

```diff
@@ -230,3 +230,13 @@ EOF
 	#then
 	wait_for_log '"updating runtime configuration"'
 }
+
+@test "reload config should update 'pinned_images'" {
+	# given
+	printf '[crio.image]\npinned_images = [""]\n' > "$CRIO_CONFIG_DIR"/00-default
+	printf '[crio.image]\npinned_images = ["quay.io/crio/hello-wasm:latest"]\n' > "$CRIO_CONFIG_DIR"/01-overwrite
+	# when
+	reload_crio
+	#then
+	wait_for_log 'Updated new pinned images'
```

Comment:
I think this test could do a `crictl images -o json` and find the pinned images, then do so after we do the reload and make sure the images are updated

Reply:
This comment led me to understand that the current implementation is wrong, and it doesn't update the pinned attribute :) 

Thanks 🙏

Reply:
I trust the config hadn't pinned anything else, but just to future proof, can you also do the opposite check before we reload_crio?